frontend/webpack-stats.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"status":"done","publicPath":"http://localhost:3000/","chunks":{"main":[{"name":"main-2b5d0e16452fb2abbd4f.js","publicPath":"http://localhost:3000/main-2b5d0e16452fb2abbd4f.js","path":"/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/edx_figures/static/edx_figures_bundles/main-2b5d0e16452fb2abbd4f.js"},{"name":"0.7d471b9f1ef73226483d.hot-update.js","publicPath":"http://localhost:3000/0.7d471b9f1ef73226483d.hot-update.js","path":"/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/edx_figures/static/edx_figures_bundles/0.7d471b9f1ef73226483d.hot-update.js"},{"name":"styles.css","publicPath":"http://localhost:3000/styles.css","path":"/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/edx_figures/static/edx_figures_bundles/styles.css"},{"name":"main-2b5d0e16452fb2abbd4f.js.map","publicPath":"http://localhost:3000/main-2b5d0e16452fb2abbd4f.js.map","path":"/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/edx_figures/static/edx_figures_bundles/main-2b5d0e16452fb2abbd4f.js.map"},{"name":"0.7d471b9f1ef73226483d.hot-update.js.map","publicPath":"http://localhost:3000/0.7d471b9f1ef73226483d.hot-update.js.map","path":"/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/edx_figures/static/edx_figures_bundles/0.7d471b9f1ef73226483d.hot-update.js.map"},{"name":"styles.css.map","publicPath":"http://localhost:3000/styles.css.map","path":"/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/edx_figures/static/edx_figures_bundles/styles.css.map"}]},"error":"ModuleBuildError","message":"Module build failed: ModuleBuildError: Module build failed: \n      @include editable-content-tooltip;\n              ^\n      No mixin named editable-content-tooltip\n\nBacktrace:\n\tstdin:89\n      in /Users/Matej/Work/Dev/ficus_figures/src/edx-figures/frontend/src/views/_single-report-content.scss (line 89, column 16)\n    at runLoaders (/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/frontend/node_modules/webpack/lib/NormalModule.js:195:19)\n    at /Users/Matej/Work/Dev/ficus_figures/src/edx-figures/frontend/node_modules/loader-runner/lib/LoaderRunner.js:364:11\n    at /Users/Matej/Work/Dev/ficus_figures/src/edx-figures/frontend/node_modules/loader-runner/lib/LoaderRunner.js:230:18\n    at context.callback (/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/frontend/node_modules/loader-runner/lib/LoaderRunner.js:111:13)\n    at Object.asyncSassJobQueue.push [as callback] (/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/frontend/node_modules/sass-loader/lib/loader.js:55:13)\n    at Object.<anonymous> (/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/frontend/node_modules/async/dist/async.js:2257:31)\n    at Object.callback (/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/frontend/node_modules/async/dist/async.js:958:16)\n    at options.error (/Users/Matej/Work/Dev/ficus_figures/src/edx-figures/frontend/node_modules/node-sass/lib/index.js:294:32)"}
```

Comment:
I made a PR global comment, but we really need to remove this from the repo and make a .gitignore rule. 

Bundling production assets for master so that assets don't need to get compiled just to run it make sense, but the only way I can get the front end to run in devstack is to run the webpack server, which dynamically generates webpack-stats.json anyway